src/main/java/pl/wrapper/parking/infrastructure/error/HandleResult.java

```diff
@@ -30,5 +28,8 @@ private static ErrorWrapper getInfoByError(Error error, String uri, HttpStatus o
                     "Wrong Parking Symbol: " + e.symbol(), onSuccess, uri, HttpStatus.NOT_FOUND);
             case ParkingError.ParkingNotFoundById e -> new ErrorWrapper(
                     "Wrong Parking Id: " + e.id(), onSuccess, uri, HttpStatus.NOT_FOUND);
+            case ParkingError.ParkingNotFoundByName e -> new ErrorWrapper(
+                    "Wrong Parking name: " + e.name(), onSuccess, uri, HttpStatus.NOT_FOUND);
```

Comment:
Let's change "Wrong" to something along the lines of "Parking of name ... not found".